test/extensions/filters/http/ext_authz/logging_test_filter.cc

```diff
@@ -63,6 +63,24 @@ class LoggingTestFilter : public Http::PassThroughFilter {
       ASSERT_NE(ext_authz_logging_info->upstreamHost(), nullptr);
       EXPECT_EQ(ext_authz_logging_info->upstreamHost()->cluster().name(), expected_cluster_name_);
     }
+
+    // Assert for CEL and accesslog field access
+    ASSERT_TRUE(ext_authz_logging_info->hasFieldSupport());
+    EXPECT_THAT(ext_authz_logging_info->getField("latency_us"),
+                testing::VariantWith<int64_t>(ext_authz_logging_info->latency().value().count()));
+    if (expect_envoy_grpc_specific_stats_) {
+      EXPECT_THAT(ext_authz_logging_info->getField("bytesSent"),
+                  testing::VariantWith<int64_t>(ext_authz_logging_info->bytesSent().value()));
+      if (expect_response_bytes_) {
+        EXPECT_THAT(ext_authz_logging_info->getField("bytesReceived"),
+                    testing::VariantWith<int64_t>(ext_authz_logging_info->bytesReceived().value()));
+      } else {
+        EXPECT_THAT(ext_authz_logging_info->getField("bytesReceived"),
```

Comment:
I don't think this case is ever run; it should be failing with the current version that calls `value()` instead of `has_value()` in the check. Please make sure this case (field is requested, but not value available) is covered.

Reply:
It does not fail because the value is actually set to 0 https://github.com/envoyproxy/envoy/blob/694fd2d9ac067792cf76d18be9e1dae0de89e382/source/extensions/filters/http/ext_authz/ext_authz.cc#L436, so the getField returns monostate (has_value is truthy while value() is falsy which is clearly misleading).